src/wolfgame.py

```diff
@@ -4766,7 +4766,7 @@ def chk_nightdone(cli):
                            "sorcerer", "hunter", "hag", "shaman", "crazed shaman",
                            "augur", "werekitten", "warlock", "piper", "wolf mystic",
                            "fallen angel", "vigilante", "doomsayer", "doomsayer", # NOT a mistake, doomsayer MUST be listed twice
-                           "prophet", "wolf shaman", "wolf shaman") # wolf shaman also must be listed twice
+                           "prophet", "wolf shaman", "wolf shaman", "lone wolf") # wolf shaman also must be listed twice Modified
```

Comment:
This "Modified" comment has no purpose there. Also please put lone wolf before both occurrences of wolf shaman, so that the comment makes sense.